### PR TITLE
docs: updated README and release concurrency

### PR DESCRIPTION
**Pull request by karl-cardenas-coding**
## Describe the Change

This PR updates the README with documentation that guides how to reference other documentation pages.

## Review Changes

💻 [Preview URL](https://github.com/spectrocloud/librarium/tree/readme-updated-9-19#markdown-links-and-urls)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -150,23 +160,69 @@ The index document for a folder follows the naming convention below. Here are so
 - Named as README (case-insensitive): `docs/Guides/README.mdx`
 - Same name as the parent folder: `docs/Guides/Guides.md`
 
-#### Referencing a page
+### Markdown Links and URLs
+
+The URL of a page will be composed of the file path to the page in context from the current file. All references to a another documentation page must end with the `.md` extension. Docusaurus will automatically remove the `.md` extension from the URL. The file path is needed for Docuasurus to generate the correct URL for the page when versioning is enabled.
```
Comment (ritawatson) [inline README.md:165]:
```suggestion
The doc page URL is composed of the file path to the page in context from the current file. All references to a another documentation page must end with the `.md` extension. Docusaurus will automatically remove the `.md` extension from the URL. The file path is needed for Docuasurus to generate the correct URL for the page when versioning is enabled.
```

File: README.md [hunk 2]
```diff
@@ -150,23 +160,69 @@ The index document for a folder follows the naming convention below. Here are so
...
 
-The url of a page will be composed from the path of the file relative to the `content` directory.
 
-**Example** docs/content/1-introduction/1-what-is.md will have http://localhost:9000/introduction/what-is as the url
 
-In markdown you can reference this page relatively to the root of the domain using this syntax:
+The following example shows how to reference a page in various scenarios. Assume you have the following folder structure when reviewing the examples below:
+
```
Comment (ritawatson) [inline README.md:170]: Really clear example/supporting text structure here.

File: README.md [hunk 3]
```diff
@@ -150,23 +160,69 @@ The index document for a folder follows the naming convention below. Here are so
...
+If you are in the file `grpc.md` and want to reference the file `iam-permissions.md`, you have to use the following syntax:
 
 ```md
-[Go to Dev Enging](/devx)
+![A list of all required IAM permissions for Palette](../aws/iam-permissions.md)
 ```
+
+#### Exceptions
+
+As of Docusarus `2.4.1`, the ability to reference pages that belong to another plugin is not available. To work around this limitation, URL  reference path must be used.
```
Comment (ritawatson) [inline README.md:218]:
```suggestion
As of Docusarus `2.4.1`, the ability to reference pages that belong to another plugin is unavailable. To work around this limitation, use a URL reference path. 
```

Comment (ritawatson) [reply README.md:218]: @karl-cardenas-coding Is a reference URL the same as a relative URL? If so, we should stick with the common term. An example would be helpful would be helpful here.

File: README.md [hunk 4]
```diff
@@ -150,23 +160,69 @@ The index document for a folder follows the naming convention below. Here are so
...
+#### Exceptions
+
+As of Docusarus `2.4.1`, the ability to reference pages that belong to another plugin is not available. To work around this limitation, URL  reference path must be used. 
+
+> [!WARNING]
+> Be aware that this approach will break versioning. The user experience will be impacted as the user will be redirected to the latest version of the page.
+
+
+In future releases, Docusaurus will support referencing pages from other plugins. Once this feature is available, we will update the documentation to reflect the changes.
+
```
Comment (ritawatson) [inline README.md:225]: @karl-cardenas-coding Two more scenarios with examples to include are referencing a subheading within the same topic (#) and in a topic in a different folder. If it's the same way as described in the above sections, then maybe just add a line in each section to reference subheadings the same way.

File: README.md [hunk 5]
```diff
@@ -150,23 +160,69 @@ The index document for a folder follows the naming convention below. Here are so
...
+        │   ├── grpc.md
+        │   └── ip-addresses.md
+        ├── aws
+        │   └── iam-permissions.md
+        ├── clusters
+        └── security.md
+```
+
+#### Same Folder
+
```
Comment (ritawatson) [inline README.md:185]: It's up to you - you could say "To ..." instead of "If you want to..." 
What you have here isn't incorrect, so it's your choice.

**Review by ritawatson — APPROVED** [review]
Really good examples. I offered a couple of suggestions.

File: README.md [hunk 6]
```diff
@@ -150,23 +160,69 @@ The index document for a folder follows the naming convention below. Here are so
...
+If you want to link to a file in the same folder, you can use the following syntax:
```
Comment (karl-cardenas-coding) [inline README.md:186]:
```suggestion
To link to a file in the same folder, you can use the following syntax:
```

**Review by ritawatson — COMMENTED** [review]
LGTM

**Pull request merged**